packages/cloud-cognitive/src/components/AddSelect/AddSelectFormControl.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {
+  RadioButton,
+  RadioButtonChecked,
+  Checkbox,
+  CheckboxChecked,
+} from '@carbon/react/icons';
+import PropTypes from 'prop-types';
+import { pkg } from '../../settings';
+import { UserProfileImage } from '../UserProfileImage';
+
+const componentName = 'AddSelectFormControl';
+const blockClass = `${pkg.prefix}--add-select__selections`;
+
+export let AddSelectFormControl = ({ item, onClick, selected, type }) => {
+  const controlProps = {
+    onClick,
+    id: item.id,
+    className: `${blockClass}-form-control-wrapper`,
+    [`aria-label`]: item.title,
```

Comment:
Should probably add `size: 20` here — icons appearing a bit smaller than in previous versions.

Current | Previous
--------|---------
![image](https://user-images.githubusercontent.com/1717398/230210795-27b34b5a-d252-4513-9371-0abc0c3467bf.png) | ![image](https://user-images.githubusercontent.com/1717398/230211293-5e734b04-20ea-4b69-9a23-e506a1562317.png)

Or was this a conscious decision to use smaller checkboxes since the icons have a thicker stroke at this size also still a bit smaller. 👀 
![image](https://user-images.githubusercontent.com/1717398/230211424-9438e431-954d-4858-a665-211f2785449e.png)